packages/react-charting/src/types/IDataPoint.ts

```diff
@@ -182,6 +182,57 @@ export interface IVerticalBarChartDataPoint {
   callOutAccessibilityData?: IAccessibilityProps;
 }
 
+export interface IHorizontalBarChartWithAxisDataPoint {
+  /**
+   * Independent value of the data point, rendered along the x-axis.
+   * If x is a number, then each y-coordinate is plotted at its x-coordinate.
+   * If x is a string, then the data is evenly spaced along the x-axis.
+   */
+  x: number;
+
+  /**
+   * Dependent value of the data point, rendered along the y-axis.
+   */
+  y: number | string;
+
+  /**
+   * Legend text for the datapoint in the chart
+   */
+  legend?: string;
+
+  /**
+   * color for the legend in the chart
+   */
+  color?: string;
+
+  /**
+   * Callout data for x axis
+   * This is an optional prop, If haven;t given legend will take
+   */
+  xAxisCalloutData?: string;
+
+  /**
+   * Callout data for y axis
+   * This is an optional prop, If haven't given data will take
+   */
+  yAxisCalloutData?: string;
+
+  /**
+   * data to render the line along with bars
+   */
+  lineData?: ILineDataInVerticalBarChart;
```

Comment:
Can you add a sample example of line along with bars

Reply:
I have removed this as we are not adding line in this chart as we discussed in requirements. So this is not required.